docs/assets/styles/_custom.scss

```diff
@@ -264,7 +264,7 @@ pre.chroma {
 /* NameBuiltinPseudo */ .chroma .bp {  }
 /* NameClass */ .chroma .nc { color: $tn-constant }
 /* NameConstant */ .chroma .no { color: $tn-blue }
-/* NameDecorator */ .chroma .nd { color: $tn-constant }
+/* NameDecorator */ .chroma .and { color: $tn-constant }
```

Comment:
revert

Reply:
That's a common issue I face with typos.

So I always triple check.

codespell provides similar results with less false positive.

Anyway, I always use them both, with languagetool too 😅